ayon_api/server_api.py

```diff
@@ -951,17 +955,18 @@ def update_event(
         summary=None,
         payload=None
     ):
-        kwargs = {}
-        for key, value in (
-            ("sender", sender),
-            ("projectName", project_name),
-            ("status", status),
-            ("description", description),
-            ("summary", summary),
-            ("payload", payload),
-        ):
-            if value is not None:
-                kwargs[key] = value
+        kwargs = {
+            key: value
+            for key, value in (
+                ("sender", sender),
+                ("projectName", project_name),
```

Comment:
Not urgent, but please use "project" instead of "projectName" here. It is at the moment aliased, but projectName is deprecated.

Reply:
```suggestion
                ("project", project_name),
```

Reply:
Done